tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -4248,8 +4245,7 @@ def get_mod(arr):
 
     def test_int(min_int, max_int, dtype):
         integer = randint(min_int, max_int-1, size=size, dtype=dtype)
-        if 0 in integer:
-            integer += 1
+        integer = np.where(integer==0, integer, 1)
```

Comment:
I had not realized `integer` was an array!!

Reply:
It wasn't  in the line you commented on. But I realised that this was wrong too ^^'

Reply:
But shouldn't this be the opposite?
```python
integer = np.where(integer == 0, 1, integer)
```
According to the [documentation](https://numpy.org/doc/stable/reference/generated/numpy.where.html):

> `numpy.where(condition[, x, y])`
> Return elements chosen from x or y depending on condition.
> Where True, yield x, otherwise yield y.

Reply:
Oops. Fixed in 3ff7a13